lms/tasks/organization.py

```diff
@@ -55,7 +55,7 @@ def schedule_monthly_deal_report(limit: int, backfill: int = 0) -> None:
                             since,
                             until,
                         )
-                        return
+                        continue
```

Comment:
I was so focus on getting the dates math right (which hopefully I got right) that didn't properly test this.